projects/plugins/jetpack/sal/class.json-api-post-jetpack.php

```diff
@@ -1,33 +1,86 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * This class extends the SAL_Post class, providing the implementation for
+ * functions that were declared in that SAL_Post class.
+ *
+ * @see WPCOM_JSON_API_Post_v1_1_Endpoint in class.wpcom-json-api-post-v1-1-endpoint.php for more context on
+ * the functions implemented here.
+ *
+ * @package automattic/jetpack
+ */
+/**
+ * Base class for Jetpack_Post.
+ */
 class Jetpack_Post extends SAL_Post {
+	/**
+	 * Defines a default value for the like counts on a post, if this hasn't been defined yet.
```

Comment:
Does these functions really all define default values, or is it that the implementation here in Jetpack (versus wpcom) doesn't have the functionality so we return dummy values for them all?

If it's the latter, I'd say something like
```suggestion
	 * Return the like counts on a post.
	 *
	 * As we don't do like counting locally, this implementation just returns 0.
```